MozziGuts_impl_RP2040.hpp

```diff
@@ -242,6 +242,14 @@ static void startAudio() {
   i2s.setBCLK(BCLK_PIN);
   i2s.setDATA(DOUT_PIN);
   i2s.setBitsPerSample(AUDIO_BITS);
+
+#if (ARDUINO_DATANOISETV_PICOADK)
+  // Soft mute and de-emphasis for audio codec
+  pinMode(25, OUTPUT);
+  digitalWrite(25, HIGH);
+  pinMode(23, OUTPUT);
+  digitalWrite(23, LOW);
+#endif
```

Comment:
I do not think it is desirable that the details of every board created are in Mozzi. Architectures yes, but if we start to "pre-config" every board that goes out this is a never ending story. Especially as what is done here can easily be done in the sketch.